graphene_plugin.py

```diff
@@ -115,6 +116,8 @@ def get_type_string_from_graphene_type( # pylint: disable=too-many-branches,too-
             return get_type_string_from_graphene_type(graphene_type.args, non_null=True)
         if graphene_type.callee.fullname == GRAPHENE_LIST_NAME:
             return wrap_in_nonnull(wrap_in_list(get_type_string_from_graphene_type(graphene_type.args)), non_null)
+        if graphene_type.callee.fullname == GRAPHENE_STRING_NAME:
+            return 'str'
         return graphene_type.callee.name
```

Comment:
I think instead of your change that specifically handles `String`, if you change this line to `graphene_type = graphene_type.callee`, it will handle the general case.

I would also include a comment above that change that says something like this:

```
        # This looks like a call to a scalar type, e.g. String().
        # Setting the type to the callee allows it to be evaluated correctly by the next check.
```

Reply:
You're absolutely right, that's a lot cleaner. I pushed 80e49ea to implement it.